ipn/ipnlocal/network-lock.go

```diff
@@ -31,6 +33,115 @@ type tkaState struct {
 	storage   *tka.FS
 }
 
+// TKASyncIfNeeded examines tka info reported from the control plane,
+// performing the steps necessary to synchronize local tka state.
+//
+// There are 4 scenarios handled here:
+//   - Enablement: nm.TKAEnabled but b.tka == nil
+//     ∴ reach out to /machine/tka/boostrap to get the genesis AUM, then
+//     initialize TKA.
+//   - Disablement: !nm.TKAEnabled but b.tka != nil
+//     ∴ reach out to /machine/tka/boostrap to read the disablement secret,
+//     then verify and clear tka local state.
+//   - Sync needed: b.tka.Head != nm.TKAHead
+//     ∴ complete multi-step synchronization flow.
+//   - Everything up to date: All other cases.
+//     ∴ chill out & crack open a cold one.
```

Comment:
let's be serious here

also, what's with the unicode?

Reply:
Its the therefore symbol!